### ISSUE TITLE
Add a non-empty `STAT` table for two-axis variable fonts

### ISSUE THREAD

**Issue by be5invis** [rsms/inter#265]
The current two-axis variable fonts (`Inter V`) doesn't have a meaningful `STAT` table, made OSes hard to identify the named instances in it.
```xml
  <STAT>
    <Version value="0x00010001"/>
    <DesignAxisRecordSize value="8"/>
    <!-- DesignAxisCount=2 -->
    <DesignAxisRecord>
      <Axis index="0">
        <AxisTag value="wght"/>
        <AxisNameID value="271"/>  <!-- Weight -->
        <AxisOrdering value="0"/>
      </Axis>
      <Axis index="1">
        <AxisTag value="slnt"/>
        <AxisNameID value="272"/>  <!-- Slant -->
        <AxisOrdering value="1"/>
      </Axis>
    </DesignAxisRecord>
    <!-- AxisValueCount=0 -->
    <ElidedFallbackNameID value="2"/>  <!-- Regular -->
  </STAT>
```

**Comment by be5invis** [comment]
This is a working `STAT` for the current Inter V:
```xml
  <STAT>
    <Version value="0x00010001"/>
    <DesignAxisRecordSize value="8"/>
    <!-- DesignAxisCount=2 -->
    <DesignAxisRecord>
      <Axis index="0">
        <AxisTag value="wght"/>
        <AxisNameID value="271"/>  <!-- Weight -->
        <AxisOrdering value="0"/>
      </Axis>
      <Axis index="1">
        <AxisTag value="slnt"/>
        <AxisNameID value="272"/>  <!-- Slant -->
        <AxisOrdering value="1"/>
      </Axis>
    </DesignAxisRecord>
    <AxisValueArray>
      <AxisValue index="0" Format="1">
        <AxisIndex value="0"/>
        <Flags value="0"/>
        <ValueNameID value="273"/>  <!-- Thin -->
        <Value value="100.0"/>
      </AxisValue>
      <AxisValue index="1" Format="1">
        <AxisIndex value="0"/>
        <Flags value="0"/>
        <ValueNameID value="275"/>  <!-- Extra Light -->
        <Value value="200.0"/>
      </AxisValue>
      <AxisValue index="2" Format="1">
        <AxisIndex value="0"/>
        <Flags value="0"/>
        <ValueNameID value="277"/>  <!-- Light -->
        <Value value="300.0"/>
      </AxisValue>
      <AxisValue index="3" Format="1">
        <AxisIndex value="0"/>
        <Flags value="2"/>
        <ValueNameID value="279"/>  <!-- Regular -->
        <Value value="400.0"/>
      </AxisValue>
      <AxisValue index="4" Format="1">
        <AxisIndex value="0"/>
        <Flags value="0"/>
        <ValueNameID value="281"/>  <!-- Medium -->
        <Value value="500.0"/>
      </AxisValue>
      <AxisValue index="5" Format="1">
        <AxisIndex value="0"/>
        <Flags value="0"/>
        <ValueNameID value="283"/>  <!-- Semi Bold -->
        <Value value="600.0"/>
      </AxisValue>
      <AxisValue index="6" Format="1">
        <AxisIndex value="0"/>
        <Flags value="0"/>
        <ValueNameID value="285"/>  <!-- Bold -->
        <Value value="700.0"/>
      </AxisValue>
      <AxisValue index="7" Format="1">
        <AxisIndex value="0"/>
        <Flags value="0"/>
        <ValueNameID value="287"/>  <!-- Extra Bold -->
        <Value value="800.0"/>
      </AxisValue>
      <AxisValue index="8" Format="1">
        <AxisIndex value="0"/>
        <Flags value="0"/>
        <ValueNameID value="289"/>  <!-- Black -->
        <Value value="900.0"/>
      </AxisValue>
      <AxisValue index="9" Format="1">
        <AxisIndex value="1"/>
        <Flags value="2"/>
        <ValueNameID value="279"/>  <!-- Regular -->
        <Value value="0"/>
      </AxisValue>
      <AxisValue index="10" Format="1">
        <AxisIndex value="1"/>
        <Flags value="0"/>
        <ValueNameID value="280"/>  <!-- Italic -->
        <Value value="-10"/>
      </AxisValue>
    </AxisValueArray>
    <ElidedFallbackNameID value="2"/>  <!-- Regular -->
  </STAT>
```

**Comment by rsms** [comment]
Thanks! Any ideas of how to incorporate this into the Glyphs file or build tooling/scripting that doesn’t involve slow transcoding with ttx?

**Comment by be5invis** [comment]
@rsms `ttx` has a `merge` mode that can bake a table into a font.

**Comment by rsms** [comment]
This has now been released as part of [version 3.14](https://github.com/rsms/inter/releases/tag/v3.14) 🎉
